bin/dlang/d2-release

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -eu
+
+# Paths
+r=$(dirname $0)/../..
+beaver=$r/bin/beaver
+
+# Use github utilities
+. $r/lib/github.sh
+
+# Convert the code to D2
+$beaver make DVER=2 d2conv
```

Comment:
I thought new stage will need to run `beaver install` again, is it safe to rely on it being cached?

Reply:
No, it's not cached, but Travis will use the global `install` command if it's not overridden.  That's the trick (in many senses :P).

Reply:
:+1: